SlashGaming-Diablo-II-Free-Resolution/src/patches/required/d2gfx_set_display_width_and_height_patch/d2gfx_set_display_width_and_height_patch_1_09d.cc

```diff
@@ -84,6 +84,16 @@ SetDisplayWidthAndHeightPatch_1_09D::GetPatchAddressAndSize01() {
           0x4BB8 - 0x4B80
       );
     }
+
+    case ::d2::GameVersion::k1_10: {
+      return PatchAddressAndSize(
+          ::mapi::GameAddress::FromOffset(
+              ::d2::DefaultLibrary::kD2GFX,
+              0x4B50
+          ),
+          0x4B89 - 0x4B50
```

Comment:
Change this ending of the patch to 0x4B88.

Reply:
Done